posthog/migrations/0799_alter_survey_type.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.2.22 on 2025-07-18 20:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0798_alter_integration_kind"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="survey",
+            name="type",
+            field=models.CharField(
+                choices=[
+                    ("popover", "popover"),
+                    ("widget", "widget"),
+                    ("button", "button"),
```

Comment:
whats this type?

Reply:
those were always there, though i don't think they're used for anything. but makes sense to remove them since we're not really using, so will do